Source/WebCore/platform/graphics/gstreamer/mse/AppendPipeline.cpp

```diff
@@ -1247,20 +1247,51 @@ static GstPadProbeReturn appendPipelinePadProbeDebugInformation(GstPad*, GstPadP
 #endif
 
 #if ENABLE(ENCRYPTED_MEDIA)
-static GstPadProbeReturn appendPipelineAppsinkPadEventProbe(GstPad*, GstPadProbeInfo* info, struct PadProbeInformation *padProbeInformation)
+void AppendPipeline::handleProtectedBufferProbeInformation(GstPadProbeInfo* info)
+{
+    if (!m_cachedProtectionEvents.size())
+        return;
+
+    GstBuffer* buffer = gst_pad_probe_info_get_buffer(info);
+    GST_DEBUG("adding %u protection events to buffer %p", m_cachedProtectionEvents.size(), buffer);
```

Comment:
We should assert or check if the probe info has a buffer, the callsites are guarded in this patch, but I reckon putting one in here as well would be a good idea